tsconfig.json

```diff
@@ -31,19 +26,18 @@
     ]
   },
   "ts-node": {
-    "require": [
-      "tsconfig-paths/register"
-    ],
+    "require": ["tsconfig-paths/register"],
     "compilerOptions": {
       "jsx": "react-jsx",
       "module": "CommonJS"
     }
   },
   "include": [
     "next-env.d.ts",
-    "**/*.ts",
-    "**/*.tsx",
-    ".storybook/preview.tsx",
+    //TODO: Revert this!
+    "src/zui/components",
+    //TODO: Revert this!
+    "src/zui/theme",
```

Comment:
This needs to be reverted before we can merge, otherwise we disable typescript in all of the program code. I tried reverting it locally and got 596 errors in 256 files. 😭